debug/requirements.txt

```diff
@@ -0,0 +1 @@
+pexpect==4.2.1
```

Comment:
Is that exact version required? I'd prefer something like >=4 or whatever. Having to change this when we discover some other distro packages 4.2.0 or something seems like a lot of work.